src/sql/azdata.d.ts

```diff
@@ -756,9 +756,7 @@ declare module 'azdata' {
 		flavor: string;
 	}
 
-	export interface QueryExecutionOptions {
-		options: Map<string, any>;
-	}
+	export type QueryExecutionOptions = { [option: string]: any };
```

Comment:
From my understanding, what was here before wasn't being properly sent to the tool service. This is a breaking API change though.

Reply:
This shouldn't need a change in my opinion. We use the same signature for other settings like SQLCMD and it works.